code/__DEFINES/atmospherics.dm

```diff
@@ -149,9 +149,9 @@
 
 //OPEN TURF ATMOS
 #define OPENTURF_DEFAULT_ATMOS		"o2=22;n2=82;TEMP=293.15" //the default air mix that open turfs spawn
-#define TCOMMS_ATMOS				"n2=100;TEMP=80" //-193,15�C telecommunications. also used for xenobiology slime killrooms
```

Comment:
please fix whatever the hell is going on here, this diff is doing literally nothing but causing spurious merge conflicts